common.mk

```diff
@@ -16,6 +16,11 @@ SHELL = /bin/bash
 OS = $(shell uname)
 IN_CI ?=
 
+ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
linke to https://github.com/mosn/htnn/pull/609#discussion_r1660050608

Here is my solution:)

@spacewander